Arrowgene.Ddon.LoginServer/Handler/GpCourseGetInfoHandler.cs

```diff
@@ -11,20 +11,17 @@
 
 namespace Arrowgene.Ddon.LoginServer.Handler
 {
-    public class GpCourseGetInfoHandler : StructurePacketHandler<LoginClient, C2LGpCourseGetInfoReq>
+    public class GpCourseGetInfoHandler : PacketHandler<LoginClient>
     {
         private static readonly ServerLogger Logger = LogProvider.Logger<ServerLogger>(typeof(GpCourseGetInfoHandler));
 
-        private AssetRepository _AssetRepo;
-
         private L2CGpCourseGetInfoRes _Response;
 
         public GpCourseGetInfoHandler(DdonLoginServer server) : base(server)
         {
-            _AssetRepo = server.AssetRepository;
             _Response = new L2CGpCourseGetInfoRes();
 
-            foreach (var Course in _AssetRepo.GPCourseInfoAsset.Courses)
+            foreach (var Course in server.AssetRepository.GPCourseInfoAsset.Courses)
```

Comment:
By doing this in the constructor, if the JSON file is updated while the server is running, the changes wouldn't apply. It'd be better to do this in the Handle method

Reply:
Sure, if we want to support hot reload we can move it there. I didn't realize that was something already supported. I was trying to not be wasteful and construct it one time. Will move it into the handler.